llvm/lib/Target/X86/GISel/X86InstructionSelector.cpp

```diff
@@ -1525,7 +1544,7 @@ bool X86InstructionSelector::materializeFP(MachineInstr &I,
 
     MachineMemOperand *MMO = MF.getMachineMemOperand(
         MachinePointerInfo::getConstantPool(MF), MachineMemOperand::MOLoad,
-        LLT::pointer(0, MF.getDataLayout().getPointerSizeInBits()), Alignment);
+        LLT::pointer(0, DataLayout.getPointerSizeInBits()), Alignment);
```

Comment:
Technically this should probably query the default global address space, but preexisting